app/views/devise/sessions/new.html.erb

```diff
@@ -22,8 +22,8 @@
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
   <title>Capital Humano en Ciencia y Tecnologia de Nuevo Leon</title>
 </head>
-<body style="background:#26a69a" class="Site">
-  <main Style="background-image: url(http://axostudent.co.uk/wp-content/uploads/2018/08/int_students-1024x737.jpg)" class="Site-content img fondo">
+<body style="background-image: url(http://axostudent.co.uk/wp-content/uploads/2018/08/int_students-1024x737.jpg)" class="Site">
```

Comment:
LGTM. Do you mind extracting this into a css file, to avoid duplication.

Reply:
Listo